bldsys/cmake/template/sample_api/CMakeLists.txt.in

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023, Arm Limited and Contributors
```

Comment:
I don't know if it's possible or not; but wouldn't it be cool if we could get the year as an auto populated field to save the developer some effort?  Also probably best to change these to something generic and let the end developer change it from there (i.e. <CHANGEME> or something similar).

Reply:
I can add a generic contributor and auto generation of year, but then original ".in" would be without license, is that ok?

Reply:
There is a problem with this header change as if I will change header to:
Copyright (c) @CURRENT_YEAR@, <<Contributor>>

Validation of CI fails:
Missing copyright:
bldsys/cmake/template/sample_api/CMakeLists.txt.in
bldsys/cmake/template/sample_api/sample.cpp.in
bldsys/cmake/template/sample_api/sample.h.in

Problem probably is with year, so I don't think that generation of this field is worth the effort.

Reply:
I have changed headers to previous version so CI passes.

Reply:
understood not worth the effort.